utils/build-script-impl

```diff
@@ -2328,8 +2496,8 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
                 esac
 
                 echo "--- Installing ${product} ---"
-                XCTEST_BUILD_DIR=$(build_directory ${deployment_target} xctest)
-                XCTEST_INSTALL_PREFIX="${INSTALL_DESTDIR}/${INSTALL_PREFIX}/lib/swift/${LIB_TARGET}"
+                XCTEST_BUILD_DIR=$(build_directory ${host} xctest)
+                XCTEST_INSTALL_PREFIX="${host_install_destdir}/${INSTALL_PREFIX}/lib/swift/${LIB_TARGET}"
```

Comment:
Sorry, I'm confused by this change.  On OS X, `host_install_destdir` is the full path to the toolchain directory on the build machine.  We then append `INSTALL_PREFIX` to it, which is the full path to the toolchain.  What was your intent behind this change?

I think we just want to append the `usr` path component, but we shouldn't hardcode that.

I think we can just keep the code the old way, what do you think?